zookeeper-server/src/main/java/org/apache/zookeeper/server/quorum/CommitProcessor.java

```diff
@@ -18,8 +18,12 @@
 
 package org.apache.zookeeper.server.quorum;
 
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
 import java.util.ArrayList;
 import java.util.LinkedList;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.concurrent.atomic.AtomicReference;
+import java.util.concurrent.LinkedBlockingQueue;
```

Comment:
Why import these 4? I don't see this used in the class, but I might be missing something.

Reply:
I had conflicts, maybe they are present on master. Will remove

Reply:
@nkalmar done